main.nf

```diff
@@ -2048,42 +2365,42 @@ process CalculateContamination {
 
 // STEP GATK MUTECT2.6 - FILTERING CALLS
 
+mutect2CallsToFilter = vcfConcatenatedForFilter.map{
+    variantCaller, idPatient, idSamplePair, vcf, tbi ->
+    [idPatient, idSamplePair, vcf, tbi]
+}.join(mergedStatsFile, by:[0,1]).join(contaminationTable, by:[0,1])
+
 process FilterMutect2Calls {
     label 'cpus_1'
 
-    tag {idSampleTN}
+    tag {idSamplePair}
 
-    publishDir "${params.outdir}/VariantCalling/${idSampleTN}/${"$variantCaller"}", mode: params.publishDirMode
+    publishDir "${params.outdir}/VariantCalling/${idSamplePair}/Mutect2", mode: params.publish_dir_mode
 
     input:
-        set variantCaller, idPatient, idSampleTN, file(unfiltered), file(unfilteredIndex) from vcfConcatenatedForFilter
-        file("${idSampleTN}.vcf.gz.stats") from mergedStatsFile
-        file("${idSampleTN}_contamination.table") from contaminationTable
+        set idPatient, idSamplePair, file(unfiltered), file(unfilteredIndex), file(stats), file(contaminationTable) from mutect2CallsToFilter
         file(dict) from ch_dict
         file(fasta) from ch_fasta
-        file(fastaFai) from ch_fastaFai
-        file(germlineResource) from ch_germlineResource
-        file(germlineResourceIndex) from ch_germlineResourceIndex
+        file(fastaFai) from ch_fai
+        file(germlineResource) from ch_germline_resource
+        file(germlineResourceIndex) from ch_germline_resource_tbi
         file(intervals) from ch_intervals
-        
-    output:
-        set val("Mutect2"), idPatient, idSampleTN,
-            file("filtered_${variantCaller}_${idSampleTN}.vcf.gz"),
-            file("filtered_${variantCaller}_${idSampleTN}.vcf.gz.tbi"),
-            file("filtered_${variantCaller}_${idSampleTN}.vcf.gz.filteringStats.tsv") into filteredMutect2Output
+      
+                  output:
+        set val("Mutect2"), idPatient, idSamplePair, file("Mutect2_filtered_${idSamplePair}.vcf.gz"), file("Mutect2_filtered_${idSamplePair}.vcf.gz.tbi"), file("Mutect2_filtered_${idSamplePair}.vcf.gz.filteringStats.tsv") into filteredMutect2Output
 
-    when: 'mutect2' in tools && params.pon
+    when: 'mutect2' in tools
```

Comment:
I read in the docs before that the filtering only happens when there is a PON, but here it always happens, so it might need fixing there

Reply:
No, actually we had that before, but we removed that behaviour.
Some people wanted filtering even without PON
Hence the warning here:
https://github.com/nf-core/sarek/pull/172/files#diff-28b7bea57b04f15beeb4c1de0ee78382R570